README.qmd

```diff
@@ -25,83 +75,133 @@ Additionally, a script has been provided to fetch some additional resources for
+
+Then, we fill in our parameter sets - this is one parameter set for each samples. Note that each sample can consist of multiple fastq files!
+
+![](assets/parameter_set_1.png)
+![](assets/parameter_set_2.png)
+
+3. Once we hit launch, we can execute the workflow by following the instructions on the screen:
 
 ``` bash
-nextflow run target/nextflow/workflows/pre_processing/main.nf \
-  -profile docker \
-  --id test \
-  --fastq_1 testData/minimal_test/input_fastq/SRR6357070_1.fastq.gz \
-  --publish_dir testData/test_output/
+cat > params.yaml <<'EOM'
+param_list:
+  - id: "WT_REP1"
+    fastq_1: [ "SRR6357070_1.fastq.gz", "SRR6357071_1.fastq.gz" ]
+    fastq_2: [ "SRR6357070_2.fastq.gz", "SRR6357071_2.fastq.gz" ]
+    strandedness: "reverse"
+  - id: "WT_REP2"
+    fastq_1: [ "SRR6357072_1.fastq.gz" ]
+    fastq_2: [ "SRR6357072_2.fastq.gz" ]
+    strandedness: "reverse"
+fasta: "testData/minimal_test/reference/genome.fasta"
+publish_dir: "full_pipeline_test/"
+gtf: "testData/minimal_test/reference/genes.gtf.gz"
+transcript_fasta: "testData/minimal_test/reference/transcriptome.fasta"
+EOM
+
+nextflow run https://packages.viash-hub.com/vsh/rnaseq.git \
+  -revision v0.2.0 \
+  -main-script target/nextflow/workflows/rnaseq/main.nf \
+  -params-file params.yaml \
+  -latest \
+  -resume
 ```
 
-Alternatively, we can run the pipeline with a sample sheet using the built-in `--param_list` functionality: (Read file paths must be specified relative to the sample sheet's path)
+### Run using Seqera Cloud
 
-``` bash
-cat > testData/minimal_test/input_fastq/sample_sheet.csv << HERE
-id,fastq_1,fastq_2,strandedness
-WT_REP1,SRR6357070_1.fastq.gz;SRR6357071_1.fastq.gz,SRR6357070_2.fastq.gz;SRR6357071_2.fastq.gz,reverse
-WT_REP2,SRR6357072_1.fastq.gz,SRR6357072_2.fastq.gz,reverse
-RAP1_UNINDUCED_REP1,SRR6357073_1.fastq.gz,,reverse
-HERE
-
-nextflow run target/nextflow/workflows/rnaseq/main.nf \
-  --param_list testData/minimal_test/input_fastq/sample_sheet.csv \
-  --publish_dir "test_results/full_pipeline_test" \
+It's also possible to run the workflow directly on [Seqera Cloud](https://cloud.seqera.io/). The required [Nextflow schema files](https://nextflow-io.github.io/nf-schema/latest/nextflow_schema/nextflow_schema_specification/) are provided with the workflows. Since Seqera Cloud does not support multiple-value parameters when using the form-based input, we will use Viash Hub to launch the [workflow](https://www.viash-hub.com/packages/rnaseq/v0.2.0/components/workflows/rnaseq). 
+
+1. First, we need to create an API token for your Seqera Cloud account.
+2. Next, we can launch the workflow by selecting `Seqera Cloud` as execution environment. Here you can add your API key, as well as the Workspace ID and Compute Environment. 
+
+![](assets/seqera_cloud_execution.png)
+
+3. We can now fill in the parameters, as described in step 2 under `Run using Nextflow`. Note that the test data need to reside on a cloud bucket for Seqera Cloud execution.
```

Comment:
Would it not be possible to use the github URLs for the test date here with nextflow?